<launch>
  <remap from="robot0/cmd_vel" to="/observer/cmd_vel" />
  <remap from="robot1/cmd_vel" to="/picket_1/cmd_vel" />
  <remap from="robot2/cmd_vel" to="/picket_2/cmd_vel" />

  <node pkg="man_joy_override" name="man_joy" type="mjo.py" />
  <node pkg="joy" name="joystick" type="joy_node" />
</launch>
